frontend/public/components/storage-class-form.tsx

```diff
@@ -171,7 +171,7 @@ class StorageClassFormWithTranslation extends React.Component<
       parameters: {
         type: {
           name: 'Type', // t('public~Type')
-          values: { io1: 'io1', gp2: 'gp2', sc1: 'sc1', st1: 'st1' },
+          values: { io1: 'io1', gp2: 'gp2', gp3: 'gp3', sc1: 'sc1', st1: 'st1' },
```

Comment:
I think the `gp3` should be first, in order to be the default

Reply:
@jhadvig The expected result for the issue in BZ didn't state `Default is gp3` for `kubernetes.io/aws-ebs`. I have updated the default for `ebs.csi.aws.com`. However I noticed the user still needs to select the default value for it to be selected. Other fields like Filesystem Type behave the same way.

Reply:
Yes, sorry you are right, on the CSI shuold default to GP3.